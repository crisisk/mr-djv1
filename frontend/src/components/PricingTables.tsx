"use client";

import { Button } from './Button';

export interface PricingFeatureSet {
  name: string;
  subtitle: string;
  price: string;
  features: string[];
  isFeatured?: boolean;
  buttonText: string;
}

export interface PricingTablesProps {
  packages?: PricingFeatureSet[];
}

const euroFormatter = new Intl.NumberFormat('nl-NL', {
  style: 'currency',
  currency: 'EUR'
});

const defaultPackages: PricingFeatureSet[] = [
  {
    name: 'Brons',
    subtitle: 'Entry-level pakket',
    price: euroFormatter.format(495),
    features: ['4 uur DJ-set', 'Basis licht- en geluidsset', 'Persoonlijk intakegesprek', 'Muziekvoorkeuren formulier'],
    buttonText: 'Meer Info',
  },
  {
    name: 'Zilver',
    subtitle: 'Meest gekozen',
    price: euroFormatter.format(795),
    features: [
      '6 uur DJ-set',
      'Uitgebreide lichtshow',
      'DJ + Saxofonist optie',
      '100% dansgarantie',
      'Onbeperkt aantal gasten',
    ],
    isFeatured: true,
    buttonText: 'Boek Nu',
  },
  {
    name: 'Goud',
    subtitle: 'Premium All-Inclusive',
    price: euroFormatter.format(1295),
    features: [
      '8 uur DJ-set',
      'Luxe licht- en geluidsset',
      'DJ + Saxofonist (inbegrepen)',
      'Ceremonie & receptie muziek',
      'Professionele apparatuur',
    ],
    buttonText: 'Vraag Offerte Aan',
  },
];

function PricingCard({ pkg }: { pkg: PricingFeatureSet }) {
  const { name, subtitle, price, features, isFeatured, buttonText } = pkg;

  const cardClasses = isFeatured
    ? 'bg-neutral-dark text-neutral-light shadow-2xl transform scale-105'
    : 'bg-neutral-light text-neutral-dark shadow-lg';

  const headerClasses = isFeatured
    ? 'text-secondary border-b border-secondary/50'
    : 'text-primary border-b border-neutral-gray-100';

  const buttonVariant = isFeatured ? 'secondary' : 'primary';

  return (
    <div className={`relative flex flex-col p-spacing-xl rounded-lg transition duration-300 ${cardClasses}`}>
      {isFeatured ? (
        <div className="absolute top-0 right-0 bg-secondary text-neutral-dark body-sm text-strong px-spacing-md py-spacing-xs rounded-tr-lg rounded-bl-lg">
          Populair
        </div>
      ) : null}
      <div className={`pb-spacing-md mb-spacing-md ${headerClasses}`}>
        <h3 className="heading-3">{name}</h3>
        <p className="body-sm opacity-80">{subtitle}</p>
      </div>
      <div className="flex items-baseline mb-spacing-lg">
        <span className="heading-1">{price}</span>
        <span className="body-md ml-spacing-xs">/ event</span>
      </div>
      <ul className="flex-grow space-y-spacing-sm mb-spacing-xl">
        {features.map((feature) => (
          <li key={feature} className="flex items-start body-md">
            <svg
              className={`w-5 h-5 mr-spacing-sm ${isFeatured ? 'text-secondary' : 'text-primary'}`}
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M5 13l4 4L19 7" />
            </svg>
            {feature}
          </li>
        ))}
      </ul>
      <Button variant={buttonVariant} size="lg" className="w-full">
        {buttonText}
      </Button>
    </div>
  );
}

export function PricingTables({ packages = defaultPackages }: PricingTablesProps) {
  return (
    <section className="py-spacing-3xl bg-neutral-gray-100">
      <div className="container mx-auto px-spacing-md">
        <h2 className="heading-2 text-center text-neutral-dark mb-spacing-2xl">Onze Pakketten</h2>
        <div className="grid grid-cols-1 md:grid-cols-3 gap-spacing-xl items-center">
          {packages.map((pkg) => (
            <PricingCard key={pkg.name} pkg={pkg} />
          ))}
        </div>
      </div>
    </section>
  );
}

export default PricingTables;
export { defaultPackages as defaultPricingPackages };
