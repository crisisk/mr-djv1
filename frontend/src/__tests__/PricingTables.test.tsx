/// <reference types="vitest" />
import React from "react";
import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import PricingTables from "../../../PricingTables.jsx";

describe("PricingTables", () => {
  it("renders event-based pricing by default", () => {
    const { asFragment } = render(<PricingTables />);

    expect(screen.getAllByText("€495")[0]).toBeInTheDocument();
    expect(screen.getByRole("button", { name: /per event/i })).toHaveAttribute(
      "aria-pressed",
      "true",
    );
    expect(asFragment()).toMatchSnapshot();
  });

  it("toggles to monthly pricing", async () => {
    const user = userEvent.setup();
    const { asFragment } = render(<PricingTables />);

    await user.click(screen.getByRole("button", { name: /per maand/i }));

    expect(screen.getAllByText("€99")[0]).toBeInTheDocument();
    expect(screen.getByRole("button", { name: /per maand/i })).toHaveAttribute(
      "aria-pressed",
      "true",
    );
    expect(asFragment()).toMatchSnapshot();
  });

  it("injects local highlights when a city slug is provided", () => {
    render(<PricingTables citySlug="eindhoven" />);

    const bronzeHighlight = screen.getByTestId("pricing-local-highlight-brons");
    const silverHighlight = screen.getByTestId("pricing-local-highlight-zilver");

    expect(bronzeHighlight).toHaveTextContent(/Evoluon/i);
    expect(silverHighlight).toHaveTextContent(/Eindhoven/i);
  });
});
