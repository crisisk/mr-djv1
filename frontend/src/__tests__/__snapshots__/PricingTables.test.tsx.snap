// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PricingTables > renders event-based pricing by default 1`] = `
<DocumentFragment>
  <section
    class="py-spacing-3xl bg-neutral-gray-100"
  >
    <div
      class="container mx-auto px-spacing-md"
    >
      <h2
        class="heading-2 text-center text-neutral-dark mb-spacing-2xl"
      >
        Onze Pakketten
      </h2>
      <div
        aria-label="Kies facturering"
        class="flex flex-col items-center mb-spacing-2xl"
        role="group"
      >
        <div
          class="inline-flex rounded-full bg-neutral-light p-spacing-xs shadow"
        >
          <button
            aria-pressed="true"
            class="px-spacing-lg py-spacing-sm rounded-full body-md transition-colors duration-300 focus:outline-none focus-visible:ring-2 focus-visible:ring-secondary bg-primary text-neutral-light"
            type="button"
          >
            Per event
          </button>
          <button
            aria-pressed="false"
            class="px-spacing-lg py-spacing-sm rounded-full body-md transition-colors duration-300 focus:outline-none focus-visible:ring-2 focus-visible:ring-secondary bg-transparent text-neutral-dark hover:bg-neutral-gray-200"
            type="button"
          >
            Per maand
          </button>
        </div>
        <p
          class="mt-spacing-sm text-center body-sm text-neutral-gray-500"
        >
          Vergelijk onze pakketten per event of kies voor een flexibel abonnement.
        </p>
      </div>
      <div
        class="grid grid-cols-1 md:grid-cols-3 gap-spacing-xl items-center"
      >
        <div
          class="relative flex flex-col p-spacing-xl rounded-lg transition duration-300 bg-neutral-light text-neutral-dark shadow-lg"
        >
          <div
            class="pb-spacing-md mb-spacing-md text-primary border-b border-neutral-gray-100"
          >
            <h3
              class="heading-3"
            >
              Brons
            </h3>
            <p
              class="body-sm opacity-80"
            >
              Entry-level pakket
            </p>
          </div>
          <div
            class="flex flex-col gap-spacing-xs mb-spacing-lg"
          >
            <div
              class="flex items-baseline"
            >
              <span
                class="heading-1"
              >
                € 495,00
              </span>
              <span
                class="body-md ml-spacing-xs"
              >
                / event
              </span>
            </div>
            <p
              class="body-sm opacity-80"
            >
              Eenmalige betaling per event.
            </p>
          </div>
          <ul
            class="flex-grow space-y-spacing-sm mb-spacing-xl"
          >
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              4 uur DJ-set
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Basis licht- en geluidsset
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Persoonlijk intakegesprek
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Muziekvoorkeuren formulier
            </li>
          </ul>
          <button
            class="inline-flex items-center justify-center rounded-md text-strong transition-colors duration-200 disabled:opacity-60 disabled:cursor-not-allowed focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 bg-primary text-neutral-light hover:bg-primary/90 focus-visible:ring-primary/40 px-spacing-lg py-spacing-md body-lg w-full"
            type="button"
          >
            Meer Info
          </button>
        </div>
        <div
          class="relative flex flex-col p-spacing-xl rounded-lg transition duration-300 bg-neutral-dark text-neutral-light shadow-2xl transform scale-105"
        >
          <div
            class="absolute top-0 right-0 bg-secondary text-neutral-dark body-sm text-strong px-spacing-md py-spacing-xs rounded-tr-lg rounded-bl-lg"
          >
            Populair
          </div>
          <div
            class="pb-spacing-md mb-spacing-md text-secondary border-b border-secondary/50"
          >
            <h3
              class="heading-3"
            >
              Zilver
            </h3>
            <p
              class="body-sm opacity-80"
            >
              Meest gekozen
            </p>
          </div>
          <div
            class="flex flex-col gap-spacing-xs mb-spacing-lg"
          >
            <div
              class="flex items-baseline"
            >
              <span
                class="heading-1"
              >
                € 795,00
              </span>
              <span
                class="body-md ml-spacing-xs"
              >
                / event
              </span>
            </div>
            <p
              class="body-sm opacity-80"
            >
              Populairste keuze voor bruiloften en bedrijfsevents.
            </p>
          </div>
          <ul
            class="flex-grow space-y-spacing-sm mb-spacing-xl"
          >
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-secondary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              6 uur DJ-set
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-secondary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Uitgebreide lichtshow
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-secondary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              DJ + Saxofonist optie
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-secondary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              100% dansgarantie
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-secondary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Onbeperkt aantal gasten
            </li>
          </ul>
          <button
            class="inline-flex items-center justify-center rounded-md text-strong transition-colors duration-200 disabled:opacity-60 disabled:cursor-not-allowed focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 bg-secondary text-neutral-dark hover:bg-secondary/90 focus-visible:ring-secondary/40 px-spacing-lg py-spacing-md body-lg w-full"
            type="button"
          >
            Boek Nu
          </button>
        </div>
        <div
          class="relative flex flex-col p-spacing-xl rounded-lg transition duration-300 bg-neutral-light text-neutral-dark shadow-lg"
        >
          <div
            class="pb-spacing-md mb-spacing-md text-primary border-b border-neutral-gray-100"
          >
            <h3
              class="heading-3"
            >
              Goud
            </h3>
            <p
              class="body-sm opacity-80"
            >
              Premium All-Inclusive
            </p>
          </div>
          <div
            class="flex flex-col gap-spacing-xs mb-spacing-lg"
          >
            <div
              class="flex items-baseline"
            >
              <span
                class="heading-1"
              >
                € 1.295,00
              </span>
              <span
                class="body-md ml-spacing-xs"
              >
                / event
              </span>
            </div>
            <p
              class="body-sm opacity-80"
            >
              All-in pakket voor luxe en grootschalige events.
            </p>
          </div>
          <ul
            class="flex-grow space-y-spacing-sm mb-spacing-xl"
          >
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              8 uur DJ-set
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Luxe licht- en geluidsset
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              DJ + Saxofonist (inbegrepen)
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Ceremonie & receptie muziek
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Professionele apparatuur
            </li>
          </ul>
          <button
            class="inline-flex items-center justify-center rounded-md text-strong transition-colors duration-200 disabled:opacity-60 disabled:cursor-not-allowed focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 bg-primary text-neutral-light hover:bg-primary/90 focus-visible:ring-primary/40 px-spacing-lg py-spacing-md body-lg w-full"
            type="button"
          >
            Vraag Offerte Aan
          </button>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;

exports[`PricingTables > toggles to monthly pricing 1`] = `
<DocumentFragment>
  <section
    class="py-spacing-3xl bg-neutral-gray-100"
  >
    <div
      class="container mx-auto px-spacing-md"
    >
      <h2
        class="heading-2 text-center text-neutral-dark mb-spacing-2xl"
      >
        Onze Pakketten
      </h2>
      <div
        aria-label="Kies facturering"
        class="flex flex-col items-center mb-spacing-2xl"
        role="group"
      >
        <div
          class="inline-flex rounded-full bg-neutral-light p-spacing-xs shadow"
        >
          <button
            aria-pressed="false"
            class="px-spacing-lg py-spacing-sm rounded-full body-md transition-colors duration-300 focus:outline-none focus-visible:ring-2 focus-visible:ring-secondary bg-transparent text-neutral-dark hover:bg-neutral-gray-200"
            type="button"
          >
            Per event
          </button>
          <button
            aria-pressed="true"
            class="px-spacing-lg py-spacing-sm rounded-full body-md transition-colors duration-300 focus:outline-none focus-visible:ring-2 focus-visible:ring-secondary bg-primary text-neutral-light"
            type="button"
          >
            Per maand
          </button>
        </div>
        <p
          class="mt-spacing-sm text-center body-sm text-neutral-gray-500"
        >
          Vergelijk onze pakketten per event of kies voor een flexibel abonnement.
        </p>
      </div>
      <div
        class="grid grid-cols-1 md:grid-cols-3 gap-spacing-xl items-center"
      >
        <div
          class="relative flex flex-col p-spacing-xl rounded-lg transition duration-300 bg-neutral-light text-neutral-dark shadow-lg"
        >
          <div
            class="pb-spacing-md mb-spacing-md text-primary border-b border-neutral-gray-100"
          >
            <h3
              class="heading-3"
            >
              Brons
            </h3>
            <p
              class="body-sm opacity-80"
            >
              Entry-level pakket
            </p>
          </div>
          <div
            class="flex flex-col gap-spacing-xs mb-spacing-lg"
          >
            <div
              class="flex items-baseline"
            >
              <span
                class="heading-1"
              >
                € 99,00
              </span>
              <span
                class="body-md ml-spacing-xs"
              >
                / maand
              </span>
            </div>
            <p
              class="body-sm opacity-80"
            >
              Flexibel maandelijks plan voor terugkerende events.
            </p>
          </div>
          <ul
            class="flex-grow space-y-spacing-sm mb-spacing-xl"
          >
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              4 uur DJ-set
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Basis licht- en geluidsset
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Persoonlijk intakegesprek
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Muziekvoorkeuren formulier
            </li>
          </ul>
          <button
            class="inline-flex items-center justify-center rounded-md text-strong transition-colors duration-200 disabled:opacity-60 disabled:cursor-not-allowed focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 bg-primary text-neutral-light hover:bg-primary/90 focus-visible:ring-primary/40 px-spacing-lg py-spacing-md body-lg w-full"
            type="button"
          >
            Meer Info
          </button>
        </div>
        <div
          class="relative flex flex-col p-spacing-xl rounded-lg transition duration-300 bg-neutral-dark text-neutral-light shadow-2xl transform scale-105"
        >
          <div
            class="absolute top-0 right-0 bg-secondary text-neutral-dark body-sm text-strong px-spacing-md py-spacing-xs rounded-tr-lg rounded-bl-lg"
          >
            Populair
          </div>
          <div
            class="pb-spacing-md mb-spacing-md text-secondary border-b border-secondary/50"
          >
            <h3
              class="heading-3"
            >
              Zilver
            </h3>
            <p
              class="body-sm opacity-80"
            >
              Meest gekozen
            </p>
          </div>
          <div
            class="flex flex-col gap-spacing-xs mb-spacing-lg"
          >
            <div
              class="flex items-baseline"
            >
              <span
                class="heading-1"
              >
                € 149,00
              </span>
              <span
                class="body-md ml-spacing-xs"
              >
                / maand
              </span>
            </div>
            <p
              class="body-sm opacity-80"
            >
              Vaste maandprijs inclusief 2 events per kwartaal.
            </p>
          </div>
          <ul
            class="flex-grow space-y-spacing-sm mb-spacing-xl"
          >
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-secondary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              6 uur DJ-set
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-secondary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Uitgebreide lichtshow
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-secondary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              DJ + Saxofonist optie
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-secondary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              100% dansgarantie
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-secondary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Onbeperkt aantal gasten
            </li>
          </ul>
          <button
            class="inline-flex items-center justify-center rounded-md text-strong transition-colors duration-200 disabled:opacity-60 disabled:cursor-not-allowed focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 bg-secondary text-neutral-dark hover:bg-secondary/90 focus-visible:ring-secondary/40 px-spacing-lg py-spacing-md body-lg w-full"
            type="button"
          >
            Boek Nu
          </button>
        </div>
        <div
          class="relative flex flex-col p-spacing-xl rounded-lg transition duration-300 bg-neutral-light text-neutral-dark shadow-lg"
        >
          <div
            class="pb-spacing-md mb-spacing-md text-primary border-b border-neutral-gray-100"
          >
            <h3
              class="heading-3"
            >
              Goud
            </h3>
            <p
              class="body-sm opacity-80"
            >
              Premium All-Inclusive
            </p>
          </div>
          <div
            class="flex flex-col gap-spacing-xs mb-spacing-lg"
          >
            <div
              class="flex items-baseline"
            >
              <span
                class="heading-1"
              >
                € 199,00
              </span>
              <span
                class="body-md ml-spacing-xs"
              >
                / maand
              </span>
            </div>
            <p
              class="body-sm opacity-80"
            >
              Premium abonnement met planning- & aftermovie-services.
            </p>
          </div>
          <ul
            class="flex-grow space-y-spacing-sm mb-spacing-xl"
          >
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              8 uur DJ-set
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Luxe licht- en geluidsset
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              DJ + Saxofonist (inbegrepen)
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Ceremonie & receptie muziek
            </li>
            <li
              class="flex items-start body-md"
            >
              <svg
                class="w-5 h-5 mr-spacing-sm text-primary"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 13l4 4L19 7"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              Professionele apparatuur
            </li>
          </ul>
          <button
            class="inline-flex items-center justify-center rounded-md text-strong transition-colors duration-200 disabled:opacity-60 disabled:cursor-not-allowed focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 bg-primary text-neutral-light hover:bg-primary/90 focus-visible:ring-primary/40 px-spacing-lg py-spacing-md body-lg w-full"
            type="button"
          >
            Vraag Offerte Aan
          </button>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
